### ISSUE TITLE
Operation.Begin throws a System.DivideByZero Exception

### ISSUE THREAD

**Issue by bfitzgerald80** [nblumhardt/serilog-timings#49]
Looks like a bug was introduced in 3.0 release when `Stopwatch.Frequency` < `TimeSpan.TicksPerSecond`.  This causes `StopwatchToTimeSpanTicks` to be 0, which in turn causes `Operation.GetTimestamp()` to be throw a divide by 0 exception.  Specifically
```
System.DivideByZeroException: Attempted to divide by zero.
   at SerilogTimings.Operation..ctor(ILogger target, String messageTemplate, Object[] args, CompletionBehaviour completionBehaviour, LogEventLevel completionLevel, LogEventLevel abandonmentLevel, Nullable`1 warningThreshold)
   at SerilogTimings.Operation.Begin(String messageTemplate, Object[] args)
```

**Comment by Arthur-Neto** [comment]
Got the same error when running some Unit Tests on Azure DevOps pipelines

**Comment by nblumhardt** [comment]
Fixed in #50

**Comment by nblumhardt** [comment]
Version `3.0.1-dev-00037` is on NuGet with this change; if anyone has a chance to test and verify that it fixes their issue, I'll queue up a patch release from main. Thanks!
